Library/Formula/multimarkdown.rb

```diff
@@ -14,25 +14,22 @@ class Multimarkdown < Formula
     sha256 "7b3b8c13b58c25cd8eae393275361d7442d47e285df34898818604ac6279cb94" => :mountain_lion
   end
 
+  depends_on "cmake" => :build
+
   conflicts_with "mtools", :because => "both install `mmd` binaries"
   conflicts_with "markdown", :because => "both install `markdown` binaries"
   conflicts_with "discount", :because => "both install `markdown` binaries"
 
   def install
-    ENV.append "CFLAGS", "-g -O3 -include GLibFacade.h"
+    system "sh", "link_git_modules"
+    system "sh", "update_git_modules"
     system "make"
-    rm_f Dir["scripts/*.bat"]
-    bin.install "multimarkdown", Dir["scripts/*"]
-    prefix.install "Support"
-  end
-
-  def caveats; <<-EOS.undent
-    Support files have been installed to:
-      #{opt_prefix}/Support
-    EOS
+    cd "build"
+    system "make"
```

Comment:
In the main documentation for multimarkdown (https://github.com/fletcher/MultiMarkdown-5#compile-from-source), this step is not taken, and the previous version of the formula didn't contain this step as well, so...